nav2_msgs/action/NavigateToPoses.action

```diff
@@ -1,5 +1,5 @@
 #goal definition
```

Comment:
NavigateToPoses or NavigateWayPoints? Former could mean that robot's pose should align with intermediate poses. The latter could have a final `PoseStamped` msg and `Point` array for waypoint.

Reply:
On a separate PR we could add feedback as waypoints are reached.